homework_2/task_4.py

```diff
@@ -0,0 +1,4 @@
+vip_room = {'place_1': 'occupied','place_2': 'occupied','place_3': 'occupied','place_4': 'occupied','place_5' : 'occupied'}
```

Comment:
I can change a number of VIP places. Please find solution where I can not do this